src/autoscaler/scalingengine/scalingengine.go

```diff
@@ -68,7 +68,9 @@ func (s *scalingEngine) Scale(appId string, trigger *models.Trigger) (*models.Ap
 		Reason:       getDynamicScalingReason(trigger),
 	}
 
-	defer s.scalingEngineDB.SaveScalingHistory(history)
+	defer func() {
+		_ = s.scalingEngineDB.SaveScalingHistory(history)
```

Comment:
hmmm should we log the error here ?

Reply:
or maybe SaveScalingHistory should log it if it does not already

Reply:
it logs it anyway

Reply:
done